### ISSUE TITLE
upgrading to 3.5.1 introduces warning about @types/tailwindcss@3.1.0

### ISSUE THREAD

**Issue by AliQ80** [nuxt-community/tailwindcss-module#516]
### Version

@nuxtjs/tailwindcss: 3.5.1
nuxt: 3.0.0-rc6

### Steps to reproduce

```bash
yarn upgrade-interactive
```

### What is Expected?

no warnings

### What is actually happening?

got this warning

```bash
warning @nuxtjs/tailwindcss > @types/tailwindcss@3.1.0: This is a stub types definition. tailwindcss provides its own type definitions, so you do not need this installed.
```

### Thoughts

should @types/tailwindcss@3.1.0 be removed in the future as a dependency for the module ? if so how do I remove it myself correctly for this version?

**Comment by rezbyte** [comment]
For anyone reading, this is fixed in the latest version (`v5.3.2`).

**Comment by AliQ80** [comment]
thank you for your efforts
